### PR TITLE
doc: update s5 document

### PR DESCRIPTION
**Pull request by wuxyintel**
For WaaG VM, the User VM name is hard coded in the
lifecycle manager, this User VM ("windows") is needed
for guest shutdown.
Note: libvirt will be used to do guest shutdown and
the WaaG VM name will be configurable in furture.

The lifecycle manager starup picture in WaaG VM is out
of data, update it in this patch.

Tracked-on: #6652
Signed-off-by: Xiangyang Wu <xiangyang.wu@intel.com>

### CODE REVIEW

**Comment by acrnsi** [comment]
Can one of the admins verify this patch?

File: doc/tutorials/enable_s5.rst [hunk 1]
```diff
@@ -290,6 +290,7 @@ How to Test
       sudo python3 ~/user_vm_shutdown.py <User VM name>
 
    .. note:: The User VM name is configured in the :file:`life_mngr.conf` of User VM.
+   .. note:: For the WaaG VM, the User VM name is "windows".
```
Comment (dbkinder) [inline doc/tutorials/enable_s5.rst:293]: Better would be to combine these two notes into one:

```suggestion
      For the WaaG VM, the User VM name is "windows".
```

**Review by dbkinder — CHANGES_REQUESTED** [review]
We really don't want folks manually editing XML files. Can the changes shown in this tutorial be made using the Configurator now (in v2.7)?

File: doc/tutorials/enable_s5.rst (hunk 1 above)
Comment (wuxyintel) [reply doc/tutorials/enable_s5.rst:293]: Thanks, will update this. BTW, I think current config tool doesn't support configure communication vuart for s5 in this document. @NanlinXie Can you help double check this? thanks!

**Review by dbkinder — APPROVED** [review]
OK, Thanks.

**Pull request merged**